flint.ui/src/store/modules/point.js

```diff
@@ -146,7 +146,7 @@ export default {
     point_stepDate: [],
     point_stepLenInYears: [],
     flag: 0, //Thiis is to check if whole Run works.
-    firstRun: true
```

Comment:
Is it necessary to change the value of the firstRun state variable from true to null.? As firstRun: true indicates that it is a first time run :)

WDYT? @Crystalsage

Reply:
That's correct. That's an oversight from my side. I've fixed it since.